src/types/animation.d.ts

```diff
@@ -1,5 +1,14 @@
 import {Chart} from './index.js';
 import {AnyObject} from './basic.js';
+import {Color} from './color.js';
+
+export type Interpolator<T> = (from: T, to: T, factor: number) => T;
```

Comment:
Maybe better will to do two types
```suggestion
export type PretypedInterpolator<T> = (from: T, to: T, factor: number) => T;

export type Interpolator = <T>(from: T, to: T, factor: number) => T;
```

to use Interpolator in AnimationsSpec, and PretypedInterpolator in InterpolatorsMap

Reply:
@dangreen reading better I have understood you are proposing 2 types but with the same signature. That means if the signature will change you have to change in 2 places instead of 1. What’s the reason or use case to have 2?

Reply:
@stockiNail note the template variable location, other is for user to specify and first is for provided interpolators that can't change type in config.

Reply:
Thank you @kurkle and @dangreen ... I didn't recognize the different definition... Apologize. I'm applying...